h5p-bildetema/src/components/ShareButton/ShareButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "common/components/Button";
+import { SuccessIcon, LinkIcon } from "common/components/Icons/Icons";
+import { useState } from "react";
+
+const ShareButton = (): JSX.Element => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  const copyMe = async (): Promise<void> => {
+    const url = window.location.href;
+    try {
+      await navigator.clipboard.writeText(url);
+      setIsCopied(true);
+      setTimeout(() => setIsCopied(false), 3000);
+    } catch (error) {
+      /* TODO: Show error message to user in for example a toast */
+    }
+  };
+
+  return (
+    <Button
+      variant="capsule"
+      onClick={() => {
+        copyMe();
+      }}
+    >
+      {isCopied ? <SuccessIcon /> : <LinkIcon />}
+      {/* TODO: Translate text */}
+      <span>{isCopied ? "Kopiert" : "Kopier"}</span>
```

Comment:
Maybe it should say "Kopier link" or "Kopier lenke", so the user it not confused with the action of copying the actual collection into a new one 🤔

Reply:
Updated to "Kopier lenke" and "Lenke kopiert"